docs/api/graphql_queries.md

```diff
@@ -0,0 +1,268 @@
+# GraphQL queries
+
+## Querying content
+
+### Get a Content item
+
+To get a specific Content item by its ID, use the `article` field:
```

Comment:
I'd specify that `article` corresponds to a Content Type. It can be a `folder` or `post` as well.